submissions/vincentNjele/Database/accident_detail.sql

```diff
@@ -0,0 +1,146 @@
+
+
+
+/* 1. Number of people who survived   */
+SELECT  count(s.survival_status) FROM survival_table s WHERE s.survival_status = 1
```

Comment:
This query is giving a wrong answer. Hint:count should be COUNT (*) as accident_survivals where blah blah blah.